test/snippet/range/views/minimiser.cpp

```diff
@@ -18,4 +19,22 @@ int main()
 
     // kmer_hash with gaps, hashes: [2,7,8,14,1], minimiser: [2,1]
     seqan3::debug_stream << (text | seqan3::views::kmer_hash(0b101_shape) | seqan3::views::minimiser(4)) << '\n';
+
+    /* Minimiser view with two ranges
+     * The second range are the hash values from the reverse complement, the second reverse is necessary to put the
+     * hash values in the correct order. For the example here:
+     * ACGTAGC | seqan3::views::complement                      => TGCATCG
+     *         | std::views::reverse                            => GCTACGT
+     *         | seqan3::views::kmer_hash(seqan3::ungapped{3})  => [39 (for GCA), 28 (for CTA), 49 (for TAC),
+     *                                                              6 (for ACG), 27 (for CGT)]
+     * "GCA" is not the reverse complement from first k-mer in "ACGTAGC", which is "ACG", but "CGT" is.
```

Comment:
```suggestion
     *         | seqan3::views::kmer_hash(seqan3::ungapped{3})  => [39 (for GCT), 28 (for CTA), 49 (for TAC),
     *                                                              6 (for ACG), 27 (for CGT)]
     * "GCT" is not the reverse complement from first k-mer in "ACGTAGC", which is "ACG", but "CGT" is.
```

Reply:
I still don't understand your argument because

```suggestion
     *         | seqan3::views::kmer_hash(seqan3::ungapped{3})  => [39 (for GCT), 28 (for CTA), 49 (for TAC),
     *                                                              6 (for ACG), 27 (for CGT)]
     *         | std::views::reverse                            => [27 (for CGT), 6 (for ACG), 49 (for TAC),
     *                                                              28 (for CTA), 39 (for GCT)]
     * "GCT" is not the reverse complement from first k-mer in "ACGTAGC", which is "ACG", but "CGT" is.
```

without the `std::views::reverse` you would get

```c++
[06,27,44,50,09]
[39,28,49,06,27]
 ----------- = 6
6 = -----------
```

and with the reverse the same result:

```c++
[06,27,44,50,09]
[27,06,49,28,39]
 ----------- = 6
6 = -----------
```
I guess the example does not show your intention.

Reply:
Yes, in the example it does not make a difference in the result, but the explanation still holds.